src/SlmLocale/Module.php

```diff
@@ -44,27 +44,14 @@
 
 use SlmLocale\Locale\Detector;
 use Zend\EventManager\EventInterface;
-use Zend\Loader\StandardAutoloader;
 use Zend\ModuleManager\Feature;
 use Zend\Mvc\MvcEvent;
 use Zend\Stdlib\ResponseInterface;
 
 class Module implements
-    Feature\AutoloaderProviderInterface,
     Feature\ConfigProviderInterface,
     Feature\BootstrapListenerInterface
 {
-    public function getAutoloaderConfig()
-    {
-        return [
-            StandardAutoloader::class => [
-                'namespaces' => [
-                    __NAMESPACE__ => __DIR__ . '/src/' . __NAMESPACE__,
-                ],
-            ],
-        ];
-    }
-
     public function getConfig()
     {
         return include __DIR__ . '/config/module.config.php';
```

Comment:
whoops forgot to update this :( and this file had zero tests...

Reply:
PITA